README.md

```diff
@@ -8,24 +10,10 @@
 
 A suite of [Helm Charts](https://helm.sh/docs) for standardized installations of SPIRE components in Kubernetes environments.
 
-## Add Helm repository
-
-```bash
-helm repo add spiffe https://spiffe.github.io/helm-charts-hardened/
-helm repo update
-```
-
-## Dependencies and Version Compatibility
-
-Unless otherwise noted in an application chart README, the following dependencies will follow these prescribed version compatibility rules.
-
-| Dependency | Supported Versions |
-|:-----------|:-------------------|
-| SPIRE      | `1.8.2`            |
-| Helm       | `3.x`              |
-| Kubernetes | `1.22+`            |
```

Comment:
Removed information duplicated in the chart readme so its easier to keep up to date, and the user doesn't have to go through the same info more then once.